sdkproject/Assets/MapboxDevTools/Editor/AtlasTemplateGenerator.cs

```diff
@@ -0,0 +1,334 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.Collections;
+using System.Collections.Generic;
+using Mapbox.Unity.MeshGeneration.Data;
+using System.IO;
+
+class AtlasTemplateGenerator : EditorWindow {
+    [MenuItem("Mapbox/Atlas Template Generator")]
+    public static void ShowWindow()
+    {
+        EditorWindow.GetWindow(typeof(AtlasTemplateGenerator));
+    }
+
+    public class PixelRect
+    {
+        public int x;
+        public int y;
+
+        public int xx;
+        public int yy;
+    }
+
+    public string m_saveFileName = "AtlasTemplate";
```

Comment:
`Private fields should be prefixed with an underscore (_privateField) and generally avoid this keyword.`